src/course-outline/section-card/SectionCard.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import CardHeader from '../card-header/CardHeader';
+import { getSectionStatus } from '../utils';
+
+const SectionCard = ({ section, children }) => {
+  const [isExpanded, setIsExpanded] = useState(true);
+
+  const {
+    displayName,
+    published,
+    releasedToStudents,
+    visibleToStaffOnly,
+    visibilityState,
+    staffOnlyMessage,
+  } = section;
+
+  const sectionStatus = getSectionStatus({
+    published,
+    releasedToStudents,
+    visibleToStaffOnly,
+    visibilityState,
+    staffOnlyMessage,
+  });
+
+  const handleExpandContent = () => {
+    setIsExpanded((prevState) => !prevState);
+  };
+
+  return (
+    <div className="section-card">
+      <CardHeader
+        title={displayName}
+        sectionStatus={sectionStatus}
+        isExpanded={isExpanded}
+        handleExpand={handleExpandContent}
+      />
+      {isExpanded && (
```

Comment:
you can use collapsible component from the paragon here
https://paragon-openedx.netlify.app/components/collapsible/

Reply:
as i understand we planned to show section highlights even isExpanded false. As i was was on the legacy

Reply:
Done